### ISSUE TITLE
HASS Daikin AC connection and HASS URL

### ISSUE THREAD

**Issue by KarelQuick** [home-assistant/home-assistant.github.io#35524]
### Feedback

It is hard , if not impossible to connect to the Daikin AC integration.
I made an developer-cloud user  at https://developer.cloud.daikineurope.com.
I have my Client ID, Client Secret and made a URI.

But then the problems comes in: HASS
In system/network/home assistant URL: you have to fill in something. No clues are given what that should be.
Then starting up the integration (Daikin AC) you have to fill in things.
host: ?
API-key: ?
Password: ?
What should be where? different combinations with https://developer.cloud.daikineurope.com, ClientID or Client Secret have no success.
Please make a step by step "configuration help" in the system where you explain, which entry is used for what and 
please help me out for now with:
What should be in Home Assistant URL?
What should be in Daikin AC, host:?
What should be in Daikin AC, API-key:?
What should be in Daikin AC, Password:?

help?!
Karel.Quick@gmail.com, Holland

### URL

https://www.home-assistant.io/integrations/daikin/

### Version

2024.10.4

**Comment by fredrike** [comment]
Have you read the documentation?

This integration is for controlling either of the devices listed in https://www.home-assistant.io/integrations/daikin/#supported-hardware using the local api on those devices.

The `Host` parameter reflects the ip-address of your unit, the rest of the parameters are for each of the different models.

<img width="602" alt="image" src="https://github.com/user-attachments/assets/159cfd65-dc7d-4eba-ab5b-532db4c109d4">

**Comment by KarelQuick** [comment]
Hmm, miss communication I suppose.
I have "units fitted with the BRP069C4x wifi adapter".
So I follow the link to "see the [Daikin Europe Developer Portal](https://developer.cloud.daikineurope.com/) for more details".

That's the reverence to Daikin developer centre.
"This affects units fitted with the BRP069C4x wifi adapter."
I do have BRP069C4x wifi adapters so what i did was:
Make account, get client ID, Client secret and fill in URI

<img width="949" alt="Daikin developer Portal" src="https://github.com/user-attachments/assets/048ddc25-e070-4480-9447-5af149c9e5f6">


But there is nowhere information on what the 'Home Assistant URL' (System/Network/Homa Assistant URL/xx) should be, so if i want to login via HASS where the API and HASS can to talk to each other.

Or: Should I use the othet Daikin integration?
I have seen on Github a "project" called Daikin Onecta.
I am confused.

**Comment by fredrike** [comment]
So, what do you think this means:

<img width="717" alt="image" src="https://github.com/user-attachments/assets/758b40cd-02d3-4313-a716-af4f1518fda2">

In plain english: "Your unit is not supporting local API hence you can not use this integration".

There might be other options in hacs, but start using the forum instead of asking for support here.

**Comment by fredrike** [comment]
@home-assistant close

**Comment by KarelQuick** [comment]
I am sorry to bother you, it will never happen again!
It is clear to me that support is not your business.
I will leave some reviews behind on the forum.
No thanks, at all!
